### PR TITLE
Acquasition fun

### PR DESCRIPTION
**Pull request by rishabh-mondal**
Implementing Maximum Entropy Acquisition for Ensemble Methods and Monte Carlo Dropout Sampling Strategies and test cases

### CODE REVIEW

**Comment by nipunbatra** [comment]
image


Acquisition is spelled wrong in the PR title. Moreover, I don't know if `fun` stands for function or something else.

**Comment by patel-zeel** [comment]
@rishabh-mondal A few initial comments:
* As I mentioned in #3, shape needs to be checked differently for Ensemble/MC and Deterministic. So, this function should also inherit `DeterministicAcquisition`.
* Implement this function with https://pytorch.org/docs/stable/generated/torch.nn.LogSoftmax.html as suggested above.
* Should we call it MaxEntropy or just Entropy? I'd go with the name `Entropy`. By default, all acquisitions are maximized.
* Tests should be as a .py file, and it should tested with pytest locally.
